index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body style="height: 100%">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <nav id="navbar">
+      <a href="#home">Home</a>
+      <a href="#work">Work</a>
+      <a href="#contact">Contact</a>
+      <a href="#about">About</a>
+    </nav>
+
+    <section id="welcome-section" style="height: 100%">
+      <h1>Welcome to my Portfolio</h1>
+    </section>
+    <section id-="projects">
```

Comment:
-It is good practice to use blank lines to separate large or logical code blocks

Reply:
Excellent job here 💯 👍